<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountcalorieWindow</class>
 <widget class="QMainWindow" name="CountcalorieWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>170</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>СЧЕТЧИК КАЛОРИЙ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/carrotico.ico</normaloff>images/carrotico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="product1label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product1lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product2label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product2lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product3label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product3lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product4label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product4lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product5label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product5lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product6label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product6lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product7label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product7lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product8label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product8lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product9lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product10lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product11lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product12lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product13lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product14lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product15lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="product9label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="product10label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="product11label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 11</string>
    </property>
   </widget>
   <widget class="QLabel" name="product12label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 12</string>
    </property>
   </widget>
   <widget class="QLabel" name="product13label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 13</string>
    </property>
   </widget>
   <widget class="QLabel" name="product14label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 14</string>
    </property>
   </widget>
   <widget class="QLabel" name="product15label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт 15</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons1label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons2label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons3label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons4label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons5label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons6label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons7label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons8label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons9label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons10label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons11label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 11</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons12label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 12</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons13label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 13</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons14label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 14</string>
    </property>
   </widget>
   <widget class="QLabel" name="tons15label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масса 15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons1lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons2lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons3lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons4lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons5lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons6lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons7lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons8lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons9lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons10lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons11lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons12lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons13lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons14lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tons15lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="returnmenu">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="counter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="calorielabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Калории</string>
    </property>
   </widget>
   <widget class="QLabel" name="proteinlabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Белки</string>
    </property>
   </widget>
   <widget class="QLabel" name="fatlabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Жиры</string>
    </property>
   </widget>
   <widget class="QLabel" name="carbohydratelabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Углеводы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="calorielineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proteinlineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fatlineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="carbohydratelineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
